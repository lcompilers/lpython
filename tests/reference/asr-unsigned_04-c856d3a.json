{
    "basename": "asr-unsigned_04-c856d3a",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/unsigned_04.py",
    "infile_hash": "d1c2f82e9578ce3f2364f4bbd3177bc0ae72357c644953418eaffe4c",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-unsigned_04-c856d3a.stderr",
    "stderr_hash": "3dcdf2e97f8c5f2816bed266587c7c3743f666cf2a4602f65e8ec9b8",
    "returncode": 2
}