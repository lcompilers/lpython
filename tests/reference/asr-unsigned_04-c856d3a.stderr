semantic error: The result of the bitnot ~ operation is negative, thus out of range for u16
 --> tests/errors/unsigned_04.py:4:5
  |
4 | i = ~i
  |     ^^ use ~i32(u) for signed result or bitnot_u16(u) for unsigned result
